src/TaskRunner/Commands/BuildCommands.php

```diff
@@ -348,9 +348,27 @@ public function buildAssets(array $options = [
         }
         $this->buildAssetsInstall($theme_dir, $allowedTaskRunners, $taskRunners, $files, $options);
         $this->buildAssetsCompile($taskRunners, $options, $theme_dir);
+        $this->copyAssetsToDist($theme_dir, $options['default-theme']);
```

Comment:
Hi @ozyrys ,
I think we need to find another way to do this, is not a good idea to do this blindly during a build-dev.
The dist folder only exists after the command toolkit:build-dist is executed.
Lets have a call to find a better way